content/en/getting_started/serverless/_index.md

```diff
@@ -0,0 +1,145 @@
+
+{{< img src="getting_started/serverless/functions_view.png" alt="Close-up of two functions" style="width:80%;">}}
+
+### Serverless Insights
+In Serverless View, the rightmost column is titled **Insights**. Datadog automatically highlights potential issues in your serverless applications, such as [high errors][3] and [high duration][4]; these issues appear in the Insights column.
+
+For your serverless sample application, Datadog has likely detected a [cold start][5]. Cold starts occur when your serverless application receives a sudden increase in traffic. This can happen if the function was previously receiving a relatively constant number of requests and abruptly started receiving more—or, as in this case, when the function was previously inactive and has been invoked for the first time.
+
+## Create an error to investigate
+
+You can intentionally cause an error by editing the `datadog-sample-entry-function` in the sample app stack.
+
+```python
+  # Entry Lambda Function Code
+  def handler(event, context):
+
+    raise Exception('Throw an error.')
+```
+
+{{< img src="getting_started/serverless/aws_error.png" alt="Close-up of two functions" style="width:80%;">}}
+
+
+Deploy this change and invoke your sample app again to see how you can investigate this error in Datadog.
+
+{{< img src="getting_started/serverless/dd_serverless_view_error.png" alt="Close-up of two functions" style="width:80%;">}}
+
+Notice that `datadog-sample-entry-function` has five errors.
+
+## Function details
+Click on your function to see more details regarding invocations and recent deployments.
+
+{{< img src="getting_started/serverless/details_error.png" alt="Close-up of two functions" style="width:80%;">}}
+
+The detailed view, as shown above, contains three graphs. You can set these to display any available metric; by default, they show three [enhanced Lambda metrics][7]: invocations, errors, and duration. 
+
+Datadog generates enhanced Lambda metrics out-of-the-box with low latency, several second granularity, and detailed metadata for cold starts and custom tags. You can also view the default [enhanced Lambda metrics dashboard][8].
+
+
+### Invocations
+The **Invocations** tab displays your function’s latest invocations. 
+
+Each invocation is associated with a trace. Click on **Open Trace** to see the trace for each invocation:
+
+{{< img src="getting_started/serverless/dd_flame_graph.png" alt="Close-up of two functions" style="width:80%;">}}
+
+The **Flame Graph** tab shows exactly what happened during the duration of this invocation, including which services had the highest percentage of the total execution time. The flame graph displays the request as it travels from APIGateway, through your `datadog-sample-entry-function`, through SNS, SQS, and finally your `datadog-sample-sqs-function`.
+
+{{< img src="getting_started/serverless/trace_map.png" alt="Close-up of two functions" style="width:80%;">}}
+
```

Comment:
We should also highlight the request/response payload captured on the lambda span, which is a great help to troubleshooting errors.